documentation/logo.md

```diff
@@ -1,77 +1,78 @@
 # Logo
 
-Tässä dokumentissa on koostettuna projektissa käytetyn logo-kielen sääntöjä.
+This document contains the rules of the Logo language that is used in this project.
+The language itself is a variant of Logo, so some rules might differ from the "usual" Logo language.
 
-## Muuttujat
+## Variables
 
-Muuttuja määritetään avainsanalla make tai tee, ja antamalla " ennen muuttujan nimeä. Muuttujan arvo annetaan muuttujan nimeämisen jälkeen. Esimerkiksi muuttujalle x annetaan arvo 42
+A variable is defined with the keyword `make` (or in Finnish `tee`) and writing `"` before the variable name. The value of a variable is given after the declaring
+of the variable's name. For example for variable x we can declare a value 42 in the following way
```

Comment:
For example for variable x we can declare a value 42 in the following way
-> Example: for variable x we can declare a value 42 in the following way
tai vaan For variable x we can declare a value 42 in the following way
ehkä helpompi lukee.